uint8 MISSION_START = 0 
uint8 MISSION_END = 1
uint8 MISSION_PAUSE = 2
uint8 MISSION_RESUME = 3